src/dlsproto/client/request/internal/GetRange.d

```diff
@@ -588,6 +603,8 @@ private scope class GetRangeHandler
         {
             while (this.waitForRecords())
             {
+                .log.trace("[{}]:[{}][RecordStream] Got a batch of records, stopped is {}.",
```

Comment:
What's the default logging level?

Reply:
Depens on how the application configures the loggers. I don't think they default to `trace`, though.